### DISCUSSION TITLE
Can i change the title of the chat application ?

### DISCUSSION THREAD

**Discussion by dpkshetty** [ggml-org/llama.cpp#9415] [Q&A]
Right now, the title of the chatbot says "llama.cpp" - Is there a option / argument to change it ?

I did check all the options supported in the readme doc and don't see a way to do it
I would like to use it from a existing llama.cpp-server docker image by passing arguments to customize the chat UI title

For eg: If I am serving say Mistral model I want to set title to "llama.cpp serving Mistral" 
Hence the need to change the title based on arguments passed

Let me know pls.

**Comment by KevinZonda** [comment]
https://github.com/ggml-org/llama.cpp/blob/aa0c461efe3603639af1a1defed2438d9c16ca0f/tools/server/webui/src/components/Header.tsx#L37

Modify this line can help, but it do needs rebuild. It could be tricky to use parameter to set such stuff, cuz the webui is generated throught vite, then embedded to the binary. See CMakeLists

https://github.com/ggml-org/llama.cpp/blob/aa0c461efe3603639af1a1defed2438d9c16ca0f/tools/server/CMakeLists.txt#L17
